source/OptimaJet.DWKit.Application/Configurator.cs

```diff
@@ -86,6 +92,18 @@ public static IDbProvider AutoDetectProvider()
                 catch (ArgumentException) { }
             }
 
+            if (provider == null)
+            {
+                try
+                {
+                    using (IDbConnection connection = new OracleConnection(DWKitRuntime.ConnectionStringData)) {}

```

Comment:
Oracle? 
Is this needed, or is this just something from their updated code that we don't need?